presto-raptor/src/main/java/com/facebook/presto/raptor/storage/organization/OrganizationSet.java

```diff
@@ -63,6 +71,11 @@ public OptionalInt getBucketNumber()
         return bucketNumber;
     }
 
+    public int getPriority()
```

Comment:
Have we thought about starvation issues using priority?

Reply:
> Have we thought about starvation issues using priority?

For data without delta, it actually has the same priority, so it will be in a FIFO queue.
So under the circumstance of being able to finish delta, we won't have starvation problem.
If delta just accumulate and never get compacted into 0, then the system would also be in unhealthy status  (unhealthy status of a system can happen with or without delta)